### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by zacharykai** [Flow-Launcher/Flow.Launcher#2969]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I am using the latest version of Flow Launcher.

### Problem Description

The settings page often freezes and doesn't resize when I move it to another screen or full-screen it or make it smaller.

### To Reproduce

1. Open the settings in the app.
2. Move the window to a new screen or try to resize it.
3. It freezes/slows down the computer.

### Flow Launcher Version

1.19.1

### Windows Build Number

22631.4112

### Error Log

<details>

```shell

Replace this line with the important log contents.

```
</details>

**Comment by Kylotan** [comment]
Same here. Task Manager reports that the Settings entry under Flow.Launcher.exe is using a full CPU core, and Process Monitor suggests that the program is - in my case, at least - continually reading Nvidia-related directories:

- C:\ProgramData\NVIDIA Corporation\Drs
- C:\ProgramData\NVIDIA Corporation\ShadowPlay
- C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_d3828c822366e497\NvCamera\

Between reading those repeatedly, it is looking for and failing to find these .dlls (as they do not exist in that location, or anywhere on my machine):

<my username>\AppData\Local\FlowLauncher\app-1.19.1\aurora.dll
<my username>\AppData\Local\FlowLauncher\app-1.19.1\jengined.dll
<my username>\AppData\Local\FlowLauncher\app-1.19.1\rendspt.dll
<my username>\AppData\Local\FlowLauncher\app-1.19.1\oculusmedium.exe
<my username>\AppData\Local\FlowLauncher\app-1.19.1\petromod_nvidia_profile_identifier.ogl
<my username>\AppData\Local\FlowLauncher\app-1.19.1\jengine.dll

This process seems to go on indefinitely (e.g. an infinite loop) with very gradual memory increases, most likely just memory fragmentation rather than a sign of it making any progress with an operation.

**Comment by jjw24** [comment]
hmm so far unable to reproduce on a single display, tried resizing + maximizing settings window.

**Comment by Kylotan** [comment]
Dual monitor here, for what it's worth, and still happens under 1.19.2. Looking for the exact same DLLs as above but now under the 1.19.2 directory, The sidebar of the settings populates but the main pane on the right stays blank.

It's worth mentioning that this doesn't happen ever time. Like the other bug I've found with Flow Launcher (stopping responding to the hotkey), it seems to be something that doesn't happen immediately but is triggered after some time or by something changing in the environment.

**Comment by zacharykai** [comment]
Yes, the issue appears to exclusively happen for me on my computer with two extra monitors. The other without the extra screens doesn't have this bug.

**Comment by j-a-s-o-n-g** [comment]
I can say the same happens on my machine. It happens very frequently, but not immediately after restarting FL. For example, I'll be doing normal computer things for several hours after last pseudo-crash, right click tray -> settings and the left will populate, the right side won't. The entire window will not be interactable (but not "not responding"). I alt+f4 to force close it, but then the tray isn't interactive. No context window on right click. I either have to force quit the program or use the in-app "Restart Flow launcher" command.

Edition	Windows 11 Home
Version	23H2
OS build	22631.4169
Experience	Windows Feature Experience Pack 1000.22700.1034.0

FL: 1.19.2

2 monitor setup.

Error logs do not contain any errors.

**Comment by Kylotan** [comment]
I found an easy repro case - on a multi-monitor setup, open Flow Launcher, and open the Settings window. Then drag it to the other monitor - it then stops processing window events and CPU usage goes up, as if in an infinite loop. Even dragging it around the screen starts to be laggy. Dragging it back to the original monitor restores normal operation.
